CODEOWNERS.txt

```diff
@@ -0,0 +1,81 @@
+# If we choose to use the codeowners functionality update the file as a .md then uncomment and update the following:
```

Comment:
This comment makes me think this is meant more as a template than as an org-wide article?

Reply:
Briefly explained here: https://docs.overture.bio/docs/standards/github#codeowners

Reply:
This is completely optional if you don't want it than we can remove it just wanted to offer all that is possible